### PR TITLE
[devtools] Performance insights documentation

### PR DESCRIPTION
**Pull request by jecfish**
Only cover how to open the panel for now.

### CODE REVIEW

File: site/en/docs/devtools/performance-insights/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Performance insight: Get actionable insights on your website's performance"
```
Comment (sofiayem) [inline site/en/docs/devtools/performance-insights/index.md:3]: Performance insight**s**

File: site/en/docs/devtools/performance-insights/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+authors:
+  - jecelynyeen
+date: 2022-03-29
+description: "Get actionable, use-case based insights on your website's performance with the Performance insight panel."
```
Comment (sofiayem) [inline site/en/docs/devtools/performance-insights/index.md:7]: Missed a hyphen: use-case-based

File: site/en/docs/devtools/performance-insights/index.md (hunk 1 above)
Comment (jecfish) [reply site/en/docs/devtools/performance-insights/index.md]: insights. Fixed 2 typos.

File: site/en/docs/devtools/performance-insights/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Performance insight: Get actionable insights on your website performance"
+authors:
+  - jecelynyeen
+date: 2022-03-29
+description: "Get actionable, use-case based insights on your website performance with the Performance insight panel."
```
Comment (sofiayem) [inline site/en/docs/devtools/performance-insights/index.md]: Missed a hyphen: use-case-based

**Review by sofiayem — APPROVED** [review]

**Pull request merged**